AirCasting/MicrophoneSession/MicrophoneManager.swift

```diff
@@ -21,86 +18,110 @@ class MicrophoneManager: ObservableObject {
     private(set) var isRecording = false
     
     private var recorder: AVAudioRecorder!
+    private(set) var audioSession = AVAudioSession.sharedInstance()
```

Comment:
It makes no sense to initialize this object whit this reference until it is needed. Pleas use `lazy var`